onnx/onnx-ml.proto

```diff
@@ -452,6 +461,8 @@ message TypeProto {
     // The type of a map.
     Map map_type = 5;
 
+    Opaque opaque_type = 100;
```

Comment:
the id should be 6 here (given it's being checked in onnx repo now).

Reply:
@linkerzhang denotation string is already 6 so we have to put 7